server/README.md

```diff
@@ -0,0 +1,37 @@
+# Backend service for SpotifySocial
```

Comment:
A lot of great changes here, @SidhaantAnand. I like the README. What's SpotifySocial?

Reply:
I edited this in a commit